toolbox/cluster/set_scale.sh

```diff
@@ -0,0 +1,23 @@
+#! /bin/bash -e
+
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+source ${THIS_DIR}/../_common.sh
+
+if [ -z "${2-}" ]; then
+    echo "USAGE: ${0} instance_type scale"
+    exit 1
+fi
+
+if [ "${1-}" ]; then
+    INSTANCE_TYPE=${1}
+    ANSIBLE_OPTS="${ANSIBLE_OPTS} -e machineset_instance_type=${1}"
+fi
+
+shift
+
+if [ "${1-}" ]; then
+    echo "Setting cluster ${INSTANCE_TYPE} machinesets to have scale '${1}'"
+    ANSIBLE_OPTS="${ANSIBLE_OPTS} -e scale=${1}"
+fi
```

Comment:
Could you add a ` roles/set_scale/default/config.yml` file to define `machineset_instance_type` and `scale`, with a comment line explaining their purpose, and setting a default value if relevant?

and at the beginning of `roles/set_scale/tasks/main.yml`, make sure that the variables are defined / populated, eg:
```
fail: ...
when: not scale
```

it's not clear to me what you are testing here with the 3 `if`, and which parameters are mandatory, which are optional.

for the mandatory one in the `USAGE`, I use `<name>` and `[name]` for optional ones

Reply:
Everything is required. The last 2 `if`s are a silly mistake, got rid of them.

Reply:
`${2:-}` is the same as `${2-}` ? .... I need to take time to learn bash extended variable support :#

Reply:
Not sure, just copied it from https://github.com/openshift-psap/ci-artifacts/blob/7d033e63dc650b37d7c634080e9511af4f3d0d1e/toolbox/cluster/scaleup.sh#L6